tasks/setup_debian.yml

```diff
@@ -10,26 +10,63 @@
         state: present
         update_cache: true
 
-    - name: Install mariadb repository key
+    - name: Ensure keyrings dir exists
+      ansible.builtin.file:
+        path: /etc/apt/keyrings
+        state: directory
+        owner: root
+        group: root
+        mode: "0755"
+
+    - name: Fetch mariadb repository key
       ansible.builtin.get_url:
-        url: "https://mariadb.org/mariadb_release_signing_key.asc"
-        dest: "/etc/apt/trusted.gpg.d/mariadb.asc"
+        url: https://mariadb.org/mariadb_release_signing_key.asc
+        dest: /etc/apt/keyrings/mariadb-archive-keyring.asc
```

Comment:
Should the role be responsible for cleaning up the previous key ? Or should this be documented in the changelogs ?